src/vs/server/node/remoteExtensionsScanner.ts

```diff
@@ -28,8 +28,8 @@ export class RemoteExtensionsScannerService implements IRemoteExtensionsScannerS
 
 	readonly _serviceBrand: undefined;
 
-	private readonly _whenBuiltinExtensionsReady = Promise.resolve();
-	private readonly _whenExtensionsReady = Promise.resolve();
+	private readonly _whenBuiltinExtensionsReady = Promise.resolve<{ extensions: Array<string | URI>; installOptions: InstallOptions }>({ extensions: [], installOptions: {} });
+	private readonly _whenExtensionsReady = Promise.resolve<{ extensions: Array<string | URI>; installOptions: InstallOptions }>({ extensions: [], installOptions: {} });
```

Comment:
Do not consider builtin extensions. Idea of built in extensions is that the server is available only with built in extensions. So return information of non builtin extensions only

Reply:
It's not considered (we don't capture any failed data for builtin extensions).  I changed the signature so that I didn't need to change how these promises were being chained below, i.e:

```
this._whenExtensionsReady = this._whenBuiltinExtensionsReady = ...
```

What should I do here instead?